_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 <img class="no-flow vertical" src="/img/screenshots/notification_11.png"/>
 </div>
-This cool contribution was brought to us in its original form by [@cool-student](https://github.com/cool-student). Sadly, we couldn't get in touch with them when we were ready to focus on this PR, so [@wb9688](https://github.com/wb9688) and [@Stypox](https://github.com/Stypox) swooped in to save the day! [@wb9688](https://github.com/wb9688) updated the code to conform to our standards and coding style, and [@Stypox](https://github.com/Stypox) did the (painstaking!) work of rebasing the code after the unified player PR was merged into the development branch.
+This cool contribution, brought to you in original form by [@cool-student](https://github.com/cool-student). When ready to focus on this PR, no means of contact could be established, so [@wb9688](https://github.com/wb9688) and [@Stypox](https://github.com/Stypox) swooped in to save the day! [@wb9688](https://github.com/wb9688) updated the code to conform to our standards and coding style, and [@Stypox](https://github.com/Stypox) did the (painstaking!) work of rebasing the code after the unified player PR was merged into the development branch.
 
 Ultimately, he took over the responsibility for this PR. He updated and polished the UI, fixed bugs, got everyone and their grandfather to test the changes, and pulled us to the finish line!
 
 ## Improvements
 
-We are not done yet with what this release brings you (you may want to take a few sips of that beverage now). If you use an environment where resizing an app is possible, NewPipe can do this now as well, thanks to [@TobiGr](https://github.com/TobiGr).
+Take this moment to sip from the nectars of your beverage. [@TobiGr](https://github.com/TobiGr) lets you resize NewPipe provided it is possible in your environment.
 
-Have you ever shared a URL to NewPipe, just to be greeted with a little "Unsupported URL" toast? Well, the app still doesn't support those kinds of URLs with this release, but at least we show you a way better error message now! It allows you to reshare the URL or just offers to open it in a browser instead. All this, thanks to [@Stypox](https://github.com/Stypox) and to [@webber-naut](https://github.com/webber-naut) for bringing our attention to this in the first place. 
+Have you ever been sandwiched between sharing a URL to NewPipe, and a little "Unsupported URL" toast? Well, those URLs are still unsupported, but now the error message is great! It allows you to reshare the URL or just open it in a web-browser instead. All this, thanks to [@Stypox](https://github.com/Stypox) and [@webber-naut](https://github.com/webber-naut) for bringing this to our attention in the first place. 
 
-Talking about sharing, when you share a live stream with someone, NewPipe doesn't add a timestamp to the URL any more. This was done by [@nmurali94](https://github.com/nmurali94). Thanks.
```

Comment:
Talking about sharing ties together, good. "live stream".? What would that be, timestamp?

Reply:
> "live stream".?

Are you asking what a live stream is?

> What would that be, timestamp?

That is when you share a video link that loads the video for the user at a specific time instead of the beginning.